west.yml

```diff
@@ -48,20 +48,20 @@ manifest:
       path: modules/debug/segger
       remote: zephyrproject
     - name: mbedtls
-      repo-path: fw-nrfconnect-mbedtls
       path: modules/crypto/mbedtls
-      revision: 9d56815aa4abf99a6e96af6ed035968add4b8cab
+      revision: 4f1e8f5a78dc08aa42a47cc1ad059cce558c26c3
+      remote: zephyrproject
     - name: mcuboot
       repo-path: fw-nrfconnect-mcuboot
-      revision: f1beb319b074115883ef1f7f1e5dbe9281e5e2d0
+      revision: 0b59f48e4dce86d0c8d4e09688feb362c7223ad3
     - name: mcumgr
-      repo-path: fw-nrfconnect-mcumgr
-      revision: f663988d35da559a37f263d369842dbce309d1fa
       path: modules/lib/mcumgr
+      revision: 84934959d2d1722a23b7e7e200191ae4a6f96168
+      remote: zephyrproject
     - name: tinycbor
-      repo-path: fw-nrfconnect-tinycbor
       path: modules/lib/tinycbor
-      revision: 86d5ed5dd544b107d2d0882961a19c2c6cb06572
+      revision: 0fc68fceacd1efc1ce809c5880c380f3d98b7b6e
```

Comment:
@de-nordic Any idea why tinycbor reference should be zephyr upsteream and not NCS fork?

Reply:
> 
> 
> @de-nordic Any idea why tinycbor reference should be zephyr upsteream and not NCS fork?

Not instantly. I am looking into this.

Reply:
This is incorrect. **There should be no outside reference**. 

What should have been done instead is update to NCS fork of tinycbor (fw-nrfconnect-tinycbor) and reference to that update.

Reply:
The diff between these two SHA's is only removing IMAGE prefixes. So I believe this is correct.

When there is no difference between NCS and zephyrproject it is policy to directly use zephyrproject.

Reply:
> 
> When there is no difference between NCS and zephyrproject it is policy to directly use zephyrproject.
> 
OK, so if this is accepted policy then OK.